src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -90,11 +92,13 @@ private static String pluralize(int count, String singular, String plural) {
 		return count == 1 ? singular : plural;
 	}
 
-	private static String nullSafeMessage(AssertionError failure) {
+	private static String nullSafeMessage(Throwable failure) {
+		if (failure == null) {
```

Comment:
I am now with you @kcooney, @marcphilipp  that adding a null is an error here. Null offers no information and would suggest a bug to users looking at the message. 

To clarify the context that led me to add this  null check was the following:
http://junit.org/junit5/docs/current/api/org/junit/platform/engine/TestExecutionResult.html#failed-java.lang.Throwable-

If having a failure with a null throwable is an Error in general, wouldn't that mean that the contract of this method is also _wrong_ and should also check for a not null precondition violation also for the Throwable parameter/field? 

Looking at this without knowing the reason this was implemented this way I would vote that it is an error. I base that on the fact that a failure without any other information would not be very helpful to users.